Sources/Plasma/FeatureLib/pfPython/cyPhysics.cpp

```diff
@@ -195,7 +195,6 @@ void cyPhysics::Warp(pyPoint3& pos)
     // create message
     PyObject* matObj = pyMatrix44::New();
     pyMatrix44* mat = pyMatrix44::ConvertFrom(matObj);
-    mat->fMatrix.IdentityMatrix();
```

Comment:
I expect the actual intent of this line was to set `mat->fMatrix = hsMatrix44::IdentityMatrix()`, since `SetTranslate` does not affect the entire matrix.

Reply:
That was my thought too. Fortunately, the `pyMatrix44` constructor calls `hsMatrix44::Resset()` for us, so this is sufficient.